### PR TITLE
Task2sms

### PR DESCRIPTION
**Pull request by VasilMitovClouway**

### CODE REVIEW

File: src/SmSender.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import java.util.regex.Pattern;
+
+/**
+ * @author Vasil Mitov (v.mitov.clouway@gmail.com)
+ */
+public class SmSender implements Validator {
```
Comment (mgenov) [inline src/SmSender.java]: The role of SmsSender should be to send messages but it validates messages. Doesn't sounds reasonable.

Comment (VasilMitovClouway) [inline src/SmSender.java]: Tried to be witty and play a game of words here,as in SMS Ender/ SMS Sender depending on what the class decides to do with the sms, either to give it to the Gateway and from there to send it, or to say that there is something wrong with the given SMS by returning false.

**Comment by VasilMitovClouway** [comment]
Made changes to the code. Added a Sender interface, changed name of SmsEnder and renamed some tests to be more informative.

**Comment by mgenov** [comment]
LGTM !

**Pull request merged**